src/h5web/vis-packs/core/complex/lineConfig.tsx

```diff
@@ -0,0 +1,48 @@
+import create from 'zustand';
+import { persist } from 'zustand/middleware';
+import { useState } from 'react';
+import type { ConfigProviderProps } from '../../models';
+import createContext from 'zustand/context';
+import { ComplexLineVisType, ComplexVisType } from './models';
+
+interface ComplexLineConfig {
+  visType: ComplexLineVisType;
+  setVisType: (visType: ComplexLineVisType) => void;
+}
```

Comment:
As the `visType` is a subset of `ComplexVisType`, I prefered to use a dedicated config for the ComplexLine.